src/Microsoft.OpenApi.OData.Reader/Operation/OperationHandler.cs

```diff
@@ -44,15 +44,15 @@ public virtual OpenApiOperation CreateOperation(ODataContext context, ODataPath
             // Security
             SetSecurity(operation);
 
+            // Parameters
```

Comment:
>// Parameters [](start = 12, length = 13)

Please add a detail comments here to maintenance. For example, we must call SetParameters ahead of ...., and Why?

Reply:
Added helpful comment.
Resolved